### PR TITLE
Added social share buttons and also generated appropriate <meta> tags :)

### LINKED ISSUE #7: Add buttons to share the page on Social Media
- Add buttons so that they are prominently visible when we land on the page.
- Try various designs and discuss them here with screenshots.
- Bonus :cake: for optimizing the meta tags so that when the page is shared on social media it appears nicely

### PR DESCRIPTION
**Pull request by ghost**
However, when the screen is small, the buttons don't align properly. Appreciated if someone can tell me how to fix that :)
Closes #7

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,6 +27,14 @@
         <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
         <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
     <![endif]-->
+    
+	<!-- META TAGS FOR THE WEBSITE -->
+	<meta name="description" content="At Breeze ’16, Shiv Nadar University\'s annual cultural fest, we take you into virtual reality, letting you delve deep into the action-packed world of video gaming. Shining computer screens and gaming consoles are the rulers of today. This year, we show you how to bring those moments in front of your screens to life. Rescue the princess with Mario, and save the world with COD, race against speed itself with NFS. Brain and brawn combine to put you on the throne of this virtual reality.">
```
Comment (rhnvrm) [inline index.html:32]: Keep this to be a one to two line description

File: index.html [hunk 2]
```diff
@@ -140,11 +145,31 @@ <h1 class="brand-heading">Breeze '16</h1>
 
     <!-- Map Section -->
     <div id="map"></div>
+    
+     <!-- Social Share Buttons -->
+        <div class="share-buttons">
+            <div class="center-wrap"><h5 style="margin: 0 auto;">Spread the word: </h5></div>
+            <ul>
+                <li>
```
Comment (rhnvrm) [inline index.html:153]: Read bootstrap on how you can do this. http://getbootstrap.com/

File: css/grayscale.css [hunk 3]
```diff
@@ -381,3 +381,82 @@ img::-moz-selection {
 body {
     webkit-tap-highlight-color: rgba(255,255,255,.2);
 }
+
+
+
+/****************************
+* Social Share Buttons
+****************************/
+.share-buttons {
```
Comment (rhnvrm) [inline css/grayscale.css:390]: Do we really need a new class for this? Use bootstrap's buttons. Also, look into fontawesome for logos of websites.

File: index.html (hunk 2 above)
Comment (mr-karan) [inline index.html:153]: don't add custom css in html please! _unless_ absolutely necessary

**Pull request closed without merging**